cluster-up/cluster/kind-k8s-sriov-1.14.2/config_sriov.sh

```diff
@@ -21,12 +22,12 @@ function wait_pods_ready {
 }
 
 function deploy_sriov_operator {
-  OPERATOR_PATH=${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/sriov-network-operator-${OPERATOR_GIT_HASH}
-  if [[ ! -d $OPERATOR_PATH ]]; then
+  operator_path=${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/sriov-network-operator-${OPERATOR_GIT_HASH}
+  if [ ! -d $operator_path ]; then
     curl -L https://github.com/openshift/sriov-network-operator/archive/${OPERATOR_GIT_HASH}/sriov-network-operator.tar.gz | tar xz -C ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/
```

Comment:
I see the download was here before this PR, but my other comment applied here as well....